pulsar-functions/secrets/src/main/java/org/apache/pulsar/functions/secretsproviderconfigurator/KubernetesSecretsProviderConfigurator.java

```diff
@@ -52,7 +52,7 @@ public String getSecretsProviderClassName(Function.FunctionDetails functionDetai
             case PYTHON:
                 return "secretsprovider.EnvironmentBasedSecretsProvider";
             case GO:
-                throw new UnsupportedOperationException();
+                return "";
```

Comment:
We need to create an issue and add a comment here to link to the issue. Otherwise, we will eventually forget to add secrets provider.

Reply:
i have created a new issue https://github.com/apache/pulsar/issues/8425